Automattic/wp-calypso
# Domains: Add domains management list breadcrumbs

### Changes proposed in this Pull Request

This PR adds the breadcrumbs section for the new domain management list page. This is part of the domain management pages redesign described in pcYYhz-m2-p2.

Desktop view screenshot:

![Annotation on 2021-10-15 at 19-54-43](https://user-images.githubusercontent.com/5324818/137562590-46b50fc0-8f3c-43e4-b234-96611b11ac08.png)

Mobile view screenshot:

![Annotation on 2021-10-15 at 19-55-36](https://user-images.githubusercontent.com/5324818/137562594-19c41cae-5cbd-4ebe-a1aa-b914e9d576c4.png)

### Testing instructions

Access the live Calypso link, go to "Upgrades > Domains", and append this feature flag to your browser's URL: `?flags=domains/management-list-redesign`. Ensure the page contains the breadcrumbs as shown in the screenshot above and that all links work as expected. The page still contains only the breadcrumbs.

This PR depends on #57038.
